### ISSUE TITLE
Enhancement request for the pile you have " AM 8.33KHz channel spacing

### ISSUE THREAD

**Issue by sivaelid** [nicsure/TD-H3-Engineering#1]
Hi Nick  

First Thanks for all your hard work all look very good so far, and a  very nice suprise  to find the work you have done and a nice interface. This is a minor enhancement  to consider. Airband is now more often using 8.33KHz Chanel spacing but not selectable. Probably Tidradio will eventually do it but just a thought, I would highlight.

Dont be put off by the silly comments  by people  on things like Microsoft warnings, They just dont understand risk  v reward. 

I like Github  anyway. 

Simon G8HAM

**Comment by nicsure** [comment]
Yes, unfortunately the baked in frequency steps don't have that, I can add it on the Station software but it wouldn't be able to be saved to the radio. Firmware mod would be needed. I do need to enhance the stepping though because fine tuning is absolutely required for USB so maybe a manual entry for frequency step would be useful?

**Comment by sivaelid** [comment]
HI Thanks for your reply and update , fully understood, see if Tidradio fix it in the future

Regards Simon G8HAM

________________________________

**Comment by nicsure** [comment]
G8HAM ? That's gotta be the best callsign going. lol

**Comment by sivaelid** [comment]
Hi  yes

Had it since  Jan 1973  very lucky to get it never going to forget it lol

________________________________

**Comment by nicsure** [comment]
This has been implemented in 0.34b, although as I mentioned, it can't be saved  to the radio, it's a local setting.
